public/static/js/projects.js

```diff
@@ -781,14 +806,33 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
     return true;
   }
 
+  $scope.patchDefinitionPopulated = function(alias) {
+    return alias && (Boolean(alias.variant) || Boolean(alias.task) || !_.isEmpty(alias.variant_tags) || !_.isEmpty(alias.tags));
+  }
+
+  $scope.aliasRemotePathPopulated = function(alias) {
+    return Boolean(alias.remote_path);
```

Comment:
A more succinct way to convert to boolean is by doing !!, or just ! if you want to invert it

Reply:
I personally prefer Boolean to !! because it's more readable.